media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -948,6 +948,31 @@ textarea:focus {
 
     .container {
         padding-top: 0;
+
+        .tasks-more {
+           background-color: #0096DD;
+           color: #FFF;
+           display: inline-block;
+           border: 0px none;
+           border-radius: 0.5em;
+           text-transform: uppercase;
+           font-weight: bold;
+           cursor: pointer;
+           transition: all 0.15s ease-in-out 0s;
+           font-size: 1.125rem;
```

Comment:
We have a mixin that takes a pixel font size value and outputs rem with a px fallback. This could be <code>.font-size(18px);</code>